README.md

```diff
@@ -44,41 +51,42 @@ Current Canopy-App components:
+- See example file inputs for variables and format (`input_variables.txt` or `testf000.nc`).
+- Canopy-App assumes the NetCDF input files are in CF-Convention and test file is based on UFS-GFSv16; recommend using double or float for real variables.
 - Canopy-App can also be run with a single point of 1D input data in a text file (e.g. `input_variables_point.txt`).
 
     **Current Canopy-App Output:** Outputs 3D canopy winds, canopy vertical/eddy diffusivity values, and
     canopy photolysis attenuation correction factors, and 2D Wind Adjustment Factor (WAF).
 
-    Namelist Option : `file_out`  Prefix string (e.g., 'test') used for output file name (Currently 1D txt output with input 1D data, or 2D NetCDF output when 2D NCF input is used, i.e., infmt_opt=0).
+    Namelist Option : `file_out`  Prefix string (e.g., 'test') used to name output file (Output is 1D txt when using input 1D data (i.e., infmt_opt=1), or is 2D NetCDF output when 2D NetCDF input is used (i.e., infmt_opt=0)).
 
 
     **Table 1. Canopy-App Required Input Variables**
 
     | Variable Name    | Variable Description and Units                    |
     | ---------------  | ------------------------------------------------- |  
-    | LAT              | Latitude  (degrees)                               |
-    | LON              | Longitude (degrees; from 0-360)                   |
-    | TIME             | Timestamp (days since YYYY-N-D 0:0:0) (NetCDF Only) |
-    | FH               | Forest canopy height (m)                          |
-    | HREF             | Reference height above canopy (m)                 |
-    | WS               | Wind speed at HREF (m/s), e.g., 10 m              |
-    | CLU              | Forest clumping index (dimensionless)             |
-    | LAI              | Leaf area index (m2/m2)                           |
-    | VTYPE            | Vegetation type (dimensionless), VIIRS Only       |
-    | FFRAC            | Forest fraction (dimensionless)                   |
-    | UST              | Friction velocity (m/s)                           |
-    | CSZ              | Cosine of the solar zenith angle (dimensionless)  |
-    | Z0               | Total surface roughness length (m)                |
-    | MOL              | Monin-Obukhov Length (m)                          |
-    | FRP              | Total Fire Radiative Power (MW/grid cell area)    |
+    | lat              | Latitude  (degrees)                               |
+    | lon              | Longitude (degrees; from 0-360)                   |
+    | time             | Timestamp (days since YYYY-N-D 0:0:0) (NetCDF Only) |
+    | fh               | Forest canopy height (m)                          |
+    | href             | Reference height above canopy (m) - 10 m          |
+    | ugrd10m          | U wind at HREF (m/s), e.g., 10 m              |
+    | vgrd10m          | V wind at HREF (m/s), e.g., 10 m              |
```

Comment:
might as well fix the `|` so it lines up

Reply:
@zmoon Thank you.  Fixed.

Reply:
The additional info is nice, but it doesn't explain why it is called `testf000.nc`. Why not just `test.nc`, or put the full date in the name?

Reply:
@zmoon Agreed completely.  I have modified input file names, README, and namelist to be more descriptive.

Reply:
oops I wrote the comment in the wrong one, but anyway thanks!